style.css

```diff
@@ -137,4 +136,97 @@ nav ul {
     fill: var(--gray-2);
     top: 50%;
     transform: translateY(-50%);
+}
+
+.img-teams {
+    width: 8em;
+}
+
+/* main */
+
+main {
+    width: 60%;
+}
+
+h1 {
+    font-size: 25px;
+    font-weight: normal;
+}
+
+.btn-gray.right-radius {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
+.filter-question .btn-gray {
+    border-radius: 0;
+}
+
+.filter-question .btn-gray:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn-gray:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
+.filter-icon-left,
+.filter-icon-righ {
+    display: flex;
+    align-items: center;
+    gap: 0.2em;
+}
+
+.filter-icon-left svg {
+    width: 0.5em;
+    fill: var(--gray-2);
+}
+
+.filter-icon-right svg {
+    width: 1em;
+    fill: var(--navy);
+}
```

Comment:
Looking at those, you shouldn't need them. They should have the same width. So you should only have one `btn-icon` style, and then simply add the icons.